hardhat.config.ts

```diff
@@ -26,11 +26,17 @@ const homestead = {
 
 const config: HardhatUserConfig = {
   solidity: {
-    version: "0.8.15",
+    version: "0.8.19",
     settings: {
       optimizer: {
         enabled: true,
-        runs: 200,
+        runs: 9999999,
```

Comment:
@anudit This is a good callout!
@sanderpick This might be a little bit of a business decision since we are still in testnet.  Maybe we should scope some work to determine the best number to use here?

Reply:
I think it makes sense to max out runs, ie, drop the cost of contract calls to lowest possible, and make deploying the contract a bit more expensive. Do you know of other considerations?

Reply:
Yep that makes sense to me.  The only consideration I can think of is that we might want to get aligned on what the reasoning is.
9999999 is probably a fine choice, but If we just want to do the absolute max I believe the number is 4294967295. @anudit was there a reason you chose 9999999?

Reply:
@sanderpick @joewagner I think Etherscan at one point had a limit of 9999999 for verification.
Some thoughts on Optimizer runs https://twitter.com/PaulRBerg/status/1627682856020451328
https://github.com/aave/gho-core/blob/8221d44caed3ebc69611aa11b28d4c6fcc6090d1/hardhat.config.ts#L40

Reply:
sounds good. i'm happy to take a best practice from uniswap here.